recipes/ckanweb-configure.rb

```diff
@@ -21,7 +21,6 @@
 # limitations under the License.
 
 include_recipe "datashades::default-configure"
-include_recipe "datashades::squid-configure"
 
```

Comment:
why did squid go away here?

Reply:
It's only used by background jobs like XLoader and Archiver, so it only applies to the batch layer.

I can restore it to both if you want, just in case we switched to synchronous validation or something.

Reply:
keep it in both, better to have piece of mind since you know our ckan config says to use proxy. and if that proxy does not exist, its doomed ;)